templates/run-node.sh.j2

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+
+# This script generate a keystore if missing and run the node
+
+{% if nim_waku_rln_keystore_active %}
+if test -f {{ nim_waku_rln_keystore_path }}; then
+  echo "keystore/keystore.json already exists. Skipping the RLN registration."
+else
+
+  ## Performs the RLN registration if the keystore.json credentials file does not exist
+  exec /usr/bin/wakunode generateRlnKeystore \
+      --rln-relay-eth-client-address={{ nim_waku_rln_relay_eth_client_address }} \
+      --rln-relay-eth-private-key={{ nim_waku_rln_account_key | mandatory }} \
+      --rln-relay-eth-contract-address={{nim_waku_rln_relay_eth_contract_address}} \
+      --rln-relay-cred-path='{{ nim_waku_rln_keystore_path }}' \
+      --rln-relay-cred-password="{{ nim_waku_rln_cred_password | mandatory }}" \
+      --rln-relay-user-message-limit=100 \
+     --execute
+fi
+{% endif %}
+
+exec /usr/bin/wakunode\
+      --config-file=/conf/config.toml
+{# WARNING: Keep sensitive information in run-node.sh. #}
+{% if nim_waku_node_key is defined %}
+      --nodekey={{ nim_waku_node_key }}
+{% endif %}
+{% if "store" in nim_waku_protocols_enabled %}
+      --store-message-db-url='{{ nim_waku_store_message_db_url }}'
+{% endif %}
+{% if nim_waku_rln_keystore_active %}
+      --rln-relay-cred-password='{{ nim_waku_rln_cred_password | mandatory }}'
+{% endif %}
```

Comment:
This is wrong. Those are argument that should be in the Docker compose. This is why there is a distinction between encrypoint and command. This is why God invented `${@}`.